package.json

```diff
@@ -6,7 +6,7 @@
     "husky": "^7.0.0"
   },
   "engines": {
-    "node": ">=16.15.0"
+    "node": "16.15.0"
```

Comment:
```suggestion
    "node": "~16.15.0"
```

Reply:
(allows patch versions ie 16.15.1 etc etc)

Reply:
good to know. Either way, it didnt work to fix the heroku thing, but I like that better than >=